okta/resource_user.go

```diff
@@ -263,29 +278,15 @@ func resourceUserRead(d *schema.ResourceData, m interface{}) error {
 
 	d.Set("status", user.Status)
 
-	err = setUserProfileAttributes(d, user)
-
-	if err != nil {
+	if err = setUserProfileAttributes(d, user); err != nil {
```

Comment:
ahhh it feels so good to compress

Reply:
There are a few other instances of this throughout the provider that I wanted to fix but held off on since it didn't make sense to include in this PR to me.  I do plan on doing this in a separate PR though.